gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceGuiceModule.java

```diff
@@ -38,11 +38,7 @@
 import javax.inject.Singleton;
 
 import org.apache.gobblin.restli.EmbeddedRestliServer;
-import org.apache.gobblin.service.modules.orchestration.DagActionStore;
 import org.apache.gobblin.runtime.api.GobblinInstanceEnvironment;
-import org.apache.gobblin.service.modules.orchestration.MultiActiveLeaseArbiter;
-import org.apache.gobblin.service.modules.orchestration.MysqlMultiActiveLeaseArbiter;
-import org.apache.gobblin.service.modules.orchestration.MysqlDagActionStore;
 import org.apache.gobblin.runtime.instance.StandardGobblinInstanceLauncher;
 import org.apache.gobblin.runtime.spec_catalog.FlowCatalog;
 import org.apache.gobblin.runtime.spec_catalog.TopologyCatalog;
```

Comment:
let's please better characterize import order for our code style.  the current two bullets are imprecise - https://gobblin.apache.org/docs/developer-guide/CodingStyle/

we ought to specify:
* where `javax.` goes
* where `lombok.` goes
* that `gobblin.` is pretty much in the past, and all replaced by `org.apache.gobblin.`
* where unit-test-specific imports belong - personally I find it easiest to read `org.junit.`, `org.testng.`, `org.mockito.` etc. *following* all "non-test" imports (of what's being tested); I've observed others following this ordering as well
* that `import static` belongs after `import` (seems common practice at least...)
* putting `java.` and `javax.` first is typical and customary, but let's either justify or else drop the seemingly arbitrary guidance to deviate from alphabetical on:
    - `java.`, `org.` (non-gobblin), `com.`, `org.apache.gobblin.`
  rather than what would seem much easier for humans to scan:
    - `java.`, `com.`, `org.` (non-gobblin), `org.apache.gobblin.`